.travis.yml

```diff
@@ -31,4 +30,7 @@ install:
   - python setup.py install
 
 script:
-  - pytest
+  - if [[ "$TASK" == "lint" ]]; then
+      pip install black && black .
```

Comment:
There's a syntax here in your bash script. I think you may need to add semicolins at the end of each line?